### DISCUSSION TITLE
Please suggest how can we use the csv data file feature while running a collection

### DISCUSSION THREAD

**Discussion by DeltaUser121** [thunderclient/thunder-client-support#1369] [Q&A]
<img width="558" alt="image" src="https://github.com/rangav/thunder-client-support/assets/139353595/e635b614-5d68-4d8b-8eb4-55adee808eb1">

How to access the csv file and its data while running a collection?
If this is needed only for one request, how do we define it at test request  level

**Comment by rangav** [comment]
Hi @DeltaUser121 

Please see the sample csv data
- variables names in the first row(same names as in Environment variables), after that values for each iteration

```
var1,varName2
mark,444
john,555
```

**Comment by rangav** [comment]
Hi is your query resolved? do you still have questions about CSV input file?

**Comment by JohnSmith1Fes123** [comment]
Hi, 
I am also interested in using CSV data files. The documentation describes that there is a default data file in the UI?

> _By default, the runner will use the data file attached in the Run Collection UI. If you would like to run using a different data file, then use the --data-file argument._

But I cannot seem to find it?
In the meantime I have applied for the CLI use.

**Reply by rangav** [reply]
@JohnSmith1Fes123 can you please confirm your subscription using below form
https://www.thunderclient.com/contact
